fuzz/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "scrypt-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2018"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+scrypt = { path = "../scrypt" }
+password-hash = { version = "0.6.0-rc.0", default-features = false, features = ["rand_core"], optional = true }
+hex = "0.4"
+base64 = "0.22.1"
```

Comment:
We have the [`base16ct`](https://docs.rs/base16ct) and [`base64ct`](https://docs.rs/base64ct) crates in our org, I would prefer to use them instead.

Reply:
Sorry, my bad, I forget to remove the unused base64 crate. Fixed now.

Reply:
Please replace the `hex` crate with `base16ct` as well.

Reply:
Fixed. Sorry for the confusion.